src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+}
+
+.logo_image {
+  padding-top: 10px;
+  margin-left: 50px;
+}
+
+.header {
+  box-sizing: border-box;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding-right: 50px;
+}
+
+.menu {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.link {
+  padding-top: 23px;
+  padding-bottom: 24px;
+  color: black;
+  line-height: 60px;
+  text-decoration: none;
+  position: relative;
+  margin-left: 20px;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.link:focus {
+  color: #00acdc;
+}
+
+.is-active {
+  box-sizing: border-box;
+  color: #00acdc;
+  margin-left: 0;
```

Comment:
what if you put an `is-active` class on laptops? let's leave here only styles that we need for active link